cryptostore/aggregator/redis.py

```diff
@@ -35,11 +35,11 @@ def read(self, exchange, dtype, pair, start=None, end=None):
         key = f'{dtype}-{exchange}-{pair}'
 
         if start and end:
-            data = self.conn.xrange(key, min=start, max=end)
+            data = [[key, self.conn.xrange(key, min=start, max=end)]]
```

Comment:
why?

Reply:
The return values of redis.xread and redis.xrange are different. The modification here makes them match. This is a bug that is discovered when the xrange code paths executes.